pallets/kensetsu/src/lib.rs

```diff
@@ -201,22 +211,11 @@ pub mod pallet {
                     cdp.collateral_amount,
                     cdp.collateral_asset_id,
                 ) {
-                    Ok(cdp_is_safe) => {
-                        if !cdp_is_safe {
-                            debug!("Liquidation of CDP {:?}", cdp_id);
-                            let call = Call::<T>::liquidate { cdp_id };
-                            if let Err(err) =
-                                SubmitTransaction::<T, Call<T>>::submit_unsigned_transaction(
-                                    call.into(),
-                                )
-                            {
-                                warn!(
-                                    "Failed in offchain_worker send liquidate(cdp_id: {:?}): {:?}",
-                                    cdp_id, err
-                                );
-                            }
-                        }
+                    Ok(true) => {
+                        debug!("CDP {:?} unsafe", cdp_id);
+                        unsafe_cdp_ids.push_back(cdp_id);
```

Comment:
You push the CDP into `unsafe_cdp_ids` when `check_cdp_is_safe` returns `true`.
It looks like you push safe CDPs into the `unsafe_cdp_ids`.